crates/ruff_python_formatter/tests/snapshots/format@expression__string.py.snap

```diff
@@ -203,7 +203,20 @@ String \"\"\"
 
 "Let's" "start" "with" "a" "simple" "example"
 
-"Let's" "start" "with" "a" "simple" "example" "now repeat after me:" "I am confident" "I am confident" "I am confident" "I am confident" "I am confident"
+(
```

Comment:
This per se is a regression because black keeps the string flat inside of `ExprStmt`. However, this is the same as that black does not try to wrap binary expressions on statement level (except if they contain a list). I decided to remove the *work around* for strings so that we have the same behavior for all expressions and either solve it holistically or not at all.

Reply:
implicit string concatenation (vs. triple quoted strings) at statement level seems like an odd choice, do you by chance have any usage examples?

Reply:
I didn't find any in the whole django project... (as expected?)